workers/ohsome_quality_analyst/api/api.py

```diff
@@ -203,68 +164,40 @@ async def _fetch_indicator(
 
 
 @app.get("/report")
-async def get_report(
-    name: ReportEnum,
-    includeSvg: bool = False,
-    bpolys: Optional[str] = None,
-    dataset: Optional[DatasetEnum] = None,
-    featureId: Optional[str] = None,
-    fidField: Optional[FidFieldEnum] = None,
-):
-    """Create a Report.
-
-    Either the parameters `dataset` and `feature id` has to be provided
-    or the parameter `bpolys` in form of a GeoJSON.
+async def get_report(parameters=Depends(ReportGETRequestModel)):
+    """Get an already calculated Report for region defined by OQT.
 
-    Depending on the input, the output is a GeoJSON Feature or
-    FeatureCollection with the indicator results.
-    The Feature properties of the input GeoJSON will be preserved
-    if they do not collide with the properties set by OQT.
+    The response is a GeoJSON Feature with the Report results as properties.
+    To request an Report for a custom AOI pease use the POST method.
```

Comment:
Where did you change something that the result can't be a FeatureCollection anymore?

Reply:
`GET` requests now supports only requests for a single feature (`dataset` and `feature_id`) which always will return a Feature. This is different for the POST requests.

Reply:
And we'll never have multiple Features as one GeoJSON Object in the database?